Sources_v3/StreamChat/Models/Attachment.swift

```diff
@@ -25,6 +41,12 @@ public typealias ChatMessageAttachment = _ChatMessageAttachment<DefaultExtraData
 ///
 @dynamicMemberLookup
 public struct _ChatMessageAttachment<ExtraData: ExtraDataTypes>: Hashable {
+    /// A unique identifier of the attachment.
+    public let id: AttachmentId
+    /// A local attachment URL
+    public let localURL: URL?
+    /// A local attachment state
+    public let localState: LocalAttachmentState
```

Comment:
I know this is a draft PR but PLEASE let's not use this style of comments :) `localURL` -> `    /// A local attachment URL` is not helpful at all :) It should explain what the variable is. For example:
```
/// When a new attachment is created, this value contains the URL of the source from which the attachment 
/// data are uploaded to the server. For already sent attachments this value is usually `nil`. This value is 
/// device-specific and is not synced with other devices.
```